test/tools/tool_differ.sh

```diff
@@ -18,6 +19,12 @@ RESULT=$(diff --strip-trailing-cr -u "$EXP" <($WINE $CMD 2>&1 | sed -e "\|^${bas
 if [ $DIFF_EXIT -ne 0 ]; then
   echo failed
   echo "$RESULT"
+  echo ""
+  echo === $1 ===
+  $WINE $CMD | cat -vn
+  echo ""
+  echo === $2 ===
+  cat -vn $2
```

Comment:
What is the idea here? We shouldn't be running the command under test a second time.